src/Azure.Functions.Cli/Helpers/AzureHelper.cs

```diff
@@ -449,11 +473,63 @@ private static async Task ArmHttpAsync(HttpMethod method, Uri uri, string access
                   }, 18, TimeSpan.FromSeconds(3)
             );
         }
+        public static async Task CreateFunctionAppOnContainerService(string accessToken, string managementURL, string subscriptionId, string resourceGroup, ContainerAppsFunctionPayload payload)
+        {
+            var url = new Uri($"{managementURL}/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Web/sites/{payload.Name}?api-version={ArmUriTemplates.FunctionAppOnContainerAppsApiVersion}");
+            ColoredConsole.WriteLine(Constants.FunctionAppDeploymentToContainerAppsMessage);
+            var response = await ArmClient.HttpInvoke(HttpMethod.Put, url, accessToken, payload);
+            if (!response.IsSuccessStatusCode)
+            {
+                try
+                {
+                    var content = response.Content.ReadAsStringAsync().Result;
```

Comment:
Is there a reason we are using `.Result` here instead of `await`?

Reply:
This was an oversight. Thanks for catching it.